third_party/externals.bzl

```diff
@@ -123,43 +126,44 @@ def register_sorbet_dependencies():
     git_repository(
         name = "com_google_absl",
         remote = "https://github.com/abseil/abseil-cpp.git",
+        # commit = "63ee2f8877915a3565c29707dba8fe4d7822596a",
```

Comment:
ditto

Reply:
Yes, sorry for leaving these in there, I'll remove them.